drivers/wifi/nrf700x/Kconfig

```diff
@@ -74,10 +74,23 @@ config NRF_WIFI_IF_AUTO_START
 
 config NRF_WIFI_PATCHES_BUILTIN
 	bool "Store nRF700x FW patches as part of the driver"
```

Comment:
as part of the nRF Wi-Fi driver

Reply:
Could you risk adding, also, "residing in the on-chip memory", as this is true for now? Or play safe and just say "as part of the nRF Wi-Fi driver code" since the whole code could be offloaded (possibly) to XIP?

Reply:
This option is used for both storing in internal flash and also for relocation, so, lets leave it as is for now.

Reply:
Fine, but add the longer name at the option help-text  as said above